Handle empty control message

danilvpetrov opened the pull request:
This PR adds logic to check and handle an empty control
message in dissolve.mdns.transport.IPv6Transport.Read() method.

danilvpetrov committed:
Handle empty control message.

This commit adds a code branch to check and handle an empty control
message in dissolve.mdns.transport.IPv6Transport.Read() method.

jmalloc commented on src/dissolve/mdns/transport/ipv6.go:
```diff
@@ -75,6 +76,13 @@ func (t *IPv6Transport) Read() (*InboundPacket, error) {
 
+	if cm == nil {
+		putBuffer(buf)
+		err := fmt.Errorf("empty control message from %s", t.Group())
```
```suggestion
		err := fmt.Errorf("empty control message from %s", src)
```

jmalloc replied:
`t.Group()` is the "multicast group" that's being listened to, `src` is the origin of the packet.

danilvpetrov committed:
Replace multicast group to src in err descr

Replace erroneously added multicast group with the source of the package in the error message.

Co-Authored-By: James Harris <mailjamesharris@gmail.com>
